### DISCUSSION TITLE
Images on landmark details page

### DISCUSSION THREAD

**Discussion by deregtervictor** [OSTA-group/osta#53] [Ideas]
At the moment we have lots of information about a landmark but it would be cool if we would also have like a visual representation of the landmark we're looking for.

**Comment by BauwenDR** [comment]
Depending on the maximum image size we decide on, it may be a good idea to only download images for the landmarks in a trip.
Images for landmarks not in a trip could still work with an url, meaning they only work while online.

**Comment by BauwenDR** [comment]
I think our best option will be to show images for a landmark using an url fetched by an extension and to have a download button in the top right where the user can chose to download them.

How do we make this system also apply to videos/ other kinds of media? Some extensions may want to have video as data.
